src/app/Dashboard/components/DashboardComponent.js

```diff
@@ -42,49 +46,80 @@ export class DashboardComponent extends Component {
     pointsEarned: PropTypes.number,
     activitiesLogged: PropTypes.number,
     fetchUserActivites: PropTypes.func,
+    loadCategories: PropTypes.func,
   };
 
   componentDidMount() {
-    const { fetchUserActivites } = this.props;
+    const { fetchUserActivites, loadCategories } = this.props;
     const token = getToken();
     const userInfo = getUserInfo(token);
     this.setState({ user: userInfo });
     fetchUserActivites(userInfo.id);
+    loadCategories();
+  }
+
+  openModalLoginPointsHandler = () => {
+    const { logPoints } = this.state;
+    this.setState({
+      logPoints: true,
+    });
+    console.log('logpoints', logPoints);
```

Comment:
What is going on here?

Reply:
@kimani-ndegwa Once a user clicks on the Log points button the openModalLoginPointsHandler function triggered which in turn sets the logPoints state to true. This is necessary for the Log points form to appear.